talker_agent/install_agent.sh

```diff
@@ -1,17 +1,27 @@
-#!/bin/sh
+#!/bin/bash
 
```

Comment:
use `set -e`. this will fail the script if any of the commands fail. including `cd ...`